spec.md

```diff
@@ -209,6 +209,17 @@ that contains both context and data).
     [RFC 2046](https://tools.ietf.org/html/rfc2046)
 * For Media Type examples see [IANA Media Types](http://www.iana.org/assignments/media-types/media-types.xhtml)
 
+### correlation-field
```

Comment:
I'm not sure about this one.
It feels it should be an attribute with the value rather than a path in the data. Reading an attribute value for correlation ID is also more practical then reading a path and then fetch the value from the data.

Having this field on the spec that is optional _and_ application/system specific does not appeal to me.

Reply:
I would think that you'd want to be able to do the correlation w/o having to look at the event data itself. So I kind of assumed that if we added info like this it would be based on something in the metadata, like the event-id.

Reply:
Sean: are you suggesting adding an explicit attribute, e.g., correlation-token or correlation-id, to the Event Context/Metadata? If so, I am fine with it. The multi-event correlation ID information can either be fetched by reading a path into the data (in this case the path string needs to be available and read from the event metadata) or reading directly from an explicitly defined correlation-token value from the event metadata. 

Doug, looks like you are thinking about the same. Correct me if I misunderstand your point. 

I will update the PR soon.

Reply:
@cathyhongzhang not quite. Correlation is a system/application concern. Not every single event will have it. As such, mandating an explicit attribute feels off. I'd think an applicaiton / system that required correlation would be a part of event medatata. And 💯% not to point back to the `data` to retrieve correlation value.